lib/ansible/modules/cloud/amazon/ses_rule_set.py

```diff
@@ -0,0 +1,169 @@
+
+
+def rule_set_exists(ses_client, name):
+    rule_sets = ses_client.list_receipt_rule_sets()['RuleSets']
+    return any([s for s in rule_sets if s['Name'].lower() == name.lower()])
+
+
+def main():
+    argument_spec = ec2_argument_spec()
+    argument_spec.update(dict(
+        name=dict(type='str', required=True),
+        state=dict(type='str', default='present', choices=['present', 'absent']),
+        is_active=dict(type='bool', default=False),
+    ))
+
+    module = AnsibleModule(argument_spec=argument_spec,
+                           supports_check_mode=True)
+
+    name = module.params.get('name').lower()
+    state = module.params.get('state').lower()
+    is_active = module.params.get('is_active')
+    check_mode = module.check_mode
+    changed = False
+
+    if not HAS_BOTO3:
+        module.fail_json(msg='Python module "boto3" is missing, please install it')
+
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+    if not region:
+        module.fail_json(msg='region must be specified')
+
+    try:
+        client = boto3_conn(module, conn_type='client', resource='ses',
+                            region=region, endpoint=ec2_url, **aws_connect_kwargs)
+    except (botocore.exceptions.ClientError, botocore.exceptions.ValidationError) as e:
+        module.fail_json(msg=str(e))
+
+    if state == 'absent':
+        # Remove the rule set if present
+        if rule_set_exists(client, name):
+            changed = True
+
+            if not check_mode:
+                try:
+                    client.delete_receipt_rule_set(
+                        RuleSetName=name)
+                except (botocore.exceptions.ParamValidationError, botocore.exceptions.ClientError) as e:
+                    module.fail_json(msg=str(e))
+
+    elif state == 'present':
+        # Add rule set if missing
+        if not rule_set_exists(client, name):
+            changed = True
+
+            if not check_mode:
+                try:
+                    client.create_receipt_rule_set(
+                        RuleSetName=name)
+                except (botocore.exceptions.ParamValidationError, botocore.exceptions.ClientError) as e:
+                    module.fail_json(msg=str(e))
+
+        # Set active if requested
+        if is_active:
+            # First determine if it's the active rule set
+            try:
+                active_name = client.describe_active_receipt_rule_set()['Metadata']['Name'].lower()
+            except KeyError:
+                # Metadata was not set meaning there is no active rule set
+                active_name = ""
+
+            if name != active_name:
+                changed = True
+
+                if not check_mode:
+                    try:
+                        client.set_active_receipt_rule_set(
+                            RuleSetName=name)
+                    except (botocore.exceptions.ParamValidationError, botocore.exceptions.ClientError) as e:
```

Comment:
See the [guidelines](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#boto3-2) for better exception handling. You'll need to handle ClientError separately to ParamValidationError as only the former has a response attribute.

So 

```
try:
    result = connection.aws_call()
except ClientError as e:
    module.fail_json(msg=e.message, exception=traceback.format_exc(),
                     **camel_dict_to_snake_dict(e.response))
except ParamValidationError as e:
    module.fail_json(msg=e.message, exception=traceback.format_exc())
```